examples/structured_2d_dgsem/elixir_euler_vortex_perk4.jl

```diff
@@ -0,0 +1,114 @@
+# We use time integration methods implemented in Trixi.jl, but we need the `CallbackSet`
+using OrdinaryDiffEq: CallbackSet
```

Comment:
This will not be necessary anymore after merging #2266 since then we export `CallbackSet` directly. Depending on which PR gets merged first, this should be fixed either here or in the other PR.